src/tests/changelist.spec.ts

```diff
@@ -59,4 +59,11 @@ describe('changelist', () => {
     document.documentElement.innerHTML = res.text
     expect(document.querySelector('table > thead')).toBeFalsy()
   })
+
+  it('renders a page when searchFields is defined', async () => {
+    const server = app.getHttpServer()
+    const res = await request(server).get(`/admin/agency/agency`)
+
+    expect(res.status).toBe(200)
```

Comment:
Not a very useful test, if there's no functionality change there's no need for a functional test :) Let's keep it, we'll put the test to check that the searchbox appears here